EventFilter/L1TRawToDigi/src/implementations_stage2/GTSetup.cc

```diff
@@ -39,8 +39,9 @@ namespace l1t {
 		    PackerFactory::get()->make("stage2::GTJetPacker"),
 		    PackerFactory::get()->make("stage2::GTTauPacker"),
 		    PackerFactory::get()->make("stage2::GlobalAlgBlkPacker"),
-		    //PackerFactory::get()->make("stage2::GlobalExtBlkPacker")
+		    PackerFactory::get()->make("stage2::GlobalExtBlkPacker")
```

Comment:
We are still not packing ExtBlk as input payload.  This is commented again on top of this commit and pushed to l1t-integration-CMSSW_8_0_9 (tag v62.1)